hack/check_urls.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# check urls via trying to connect to hugo server on localhost:1313
+
+rm -rf kps
+git clone --depth 1 https://github.com/prometheus-community/helm-charts/ kps
+
+URL_PATHS=$(grep -R "runbook_url:" kps/charts/kube-prometheus-stack | grep .yaml | awk '{print $5}' | tr -d "}" | sort | uniq)
+
+ENDPOINT=${ENDPOINT:-"http://localhost:1313/runbooks"}
+
+exit_code=0
+echo "Checking runbook urls... please wait"
+
+for runbook in $URL_PATHS; do
+  if ! curl --output /dev/null --silent --head --fail "${ENDPOINT}${runbook}"; then
+    rule_file=$(grep -n -R "$runbook" kps/charts/kube-prometheus-stack | awk '{print $1}')
+    echo -e "Missing ${runbook} \n\t $EDITOR $rule_file"
+    exit_code=1
+  fi
+done
+
+if [ $exit_code == 0 ]; then
+  echo "Great Success, no issues found!"
+else
+  echo "Oh no, errors!"
+fi
+exit $exit_code
```

Comment:
We are using `mdox` for this in [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/blob/main/Makefile#L29-L37) and [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator/blob/main/Makefile#L282-L293). I don't see any reason we shouldn't be using it here too.